### DISCUSSION TITLE
Editing text input in non-title column does not allow space/backspace

### DISCUSSION THREAD

**Discussion by vozvik** [mar10/wunderbaum#13] [General]
Hi Martin,

I tried adding editable text fields into a grid following below demo and adding space when entering data does not seem to work. In my local version space (and any navigation key) is ignored, in the online demo below (column "Remarks") pressing space will reset contents:
https://mar10.github.io/wunderbaum/demo/#demo-editable

Is this wb-specific behaviour or something more generic?

Thanks!

Cheers
Viktor

**Comment by mar10** [comment]
I transferred this to issue #14

**Comment by mar10** [comment]
Closing this as completed (see #14)
